framework/tests/aliasing/UniqueConstructorTest.java

```diff
@@ -0,0 +1,16 @@
+import org.checkerframework.common.aliasing.qual.Unique;
+
+class ParentUniqueClass {
+
+    // No need for @SuppressWarnings()
+    @Unique ParentUniqueClass() {
+        // Doesn't raise unique.leaked error since it's a parent class (top of heirarchy)
+    }
+}
+
+class ChildUniqueClass extends ParentUniqueClass {
+
+    @Unique ChildUniqueClass() {
```

Comment:
Can you add another subclass that does raise an error?

Reply:
In my latest commit, the ChildUniqueClass will raise an error since the ParentClass is not @Unique.